ios/swift/AmpliSwiftSampleApp/Shared/ContentView.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ContentView.swift
+//  Shared
+//
+//  Created by Qingzhuo Zhen on 11/12/21.
+//
+
+import SwiftUI
+import CoreData
+
+struct ContentView: View {
+    @Environment(\.managedObjectContext) private var viewContext
+
+    @FetchRequest(
+        sortDescriptors: [NSSortDescriptor(keyPath: \Item.timestamp, ascending: true)],
+        animation: .default)
+    private var items: FetchedResults<Item>
+    private var ampli: Ampli = Ampli()
+    
+
+    var body: some View {
+        NavigationView {
+            List {
+                ForEach(items) { item in
+                    NavigationLink {
+                        Text("Item at \(item.timestamp!, formatter: itemFormatter)")
+                    } label: {
+                        Text(item.timestamp!, formatter: itemFormatter)
+                    }
+                }
+                .onDelete(perform: deleteItems)
+            }
+            .toolbar {
+#if os(iOS)
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    EditButton()
+                }
+#endif
+                ToolbarItem {
+                    Button(action: addItem) {
+                        Label("Add Item With Event Sent", systemImage: "plus")
+                    }
+                }
+            }
+            Text("Select an item")
+        }
+    }
+
+    private func addItem() {
+        //    Start by calling ampli.load()
+        //
+        //    'ampli' is the default instance of Ampli()
+        //
+        //    When you pull your tracking plan you can use the defaults and call load() without arguments
+        //    This requires connecting your account via `ampli pull` which will set you API key in the generated Ampli SDK
+        //
+        //    OR Specify a AmpliEnvironment
+        //    ampli.load(options: LoadOptions(environment: AmpliEnvironment.development)
+        //
+        //    OR Provide a specific Amplitude API key
+        //    ampli.load(options: LoadOptions(client: LoadClient(apiKey: "Custom api key"))
+        //
+        //    OR Use an existing Amplitude instance
+        //    requires "import Amplitude"
+        //    let instance = Amplitude.instance("instanceName");
+        //    instance.initializeApiKey("Custom api key");
+        //    ampli.load(options: LoadOptions(client: LoadClient(instance: instance)))
+        //
+        //    For testing you can disable ampli
+        //    ampli.load(options: LoadOptions(disabled: ENV.IS_TESTING ? true: false))
+        //
+        //    Make as many Ampli instances as you want
+        //    let ampli2 = new Ampli();
+        //    ampli2.load(options: LoadOptions(client: LoadClient(apiKey: "api-key-2"))
+        ampli.load(options: LoadOptions(environment: AmpliEnvironment.development, disabled: false, client: LoadClient(apiKey: ApiKey[AmpliEnvironment.development], instance: nil)));
```

Comment:
Are environment and disabled required here? I see they aren't in the examples above.

Reply:
updated. This was created before I marked the options with default